internal/registry/options.go

```diff
@@ -64,8 +64,8 @@ func (r *AuthOptionsGetter) GetOptions(ctx context.Context, repo *imagev1.ImageR
 	// Load proxy configuration.
 	var proxyURL *url.URL
 	var err error
-	if repo.Spec.ProxySecretRef != nil {
-		proxyURL, err = r.GetProxyURL(ctx, repo)
+	if repo.Spec.ProxySecretRef != nil && repo.Spec.ProxySecretRef.Name != "" {
```

Comment:
The `Name` field is required, we can keep checking only for `ProxySecretRef`

Reply:
Fixed via 046a0a9.